### PR TITLE
SSH access to Azure Arc-enabled servers

### PR DESCRIPTION
**Pull request by lanicolas**
New scenario highlighting how to connect to an Azure Arc-enabled server via SSH

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+## Prerequisites
+
+- Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+
+- [Install or update Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest). Azure CLI should be running version 1.1.0** or later. Use ```az --version``` to check your current installed version.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md:36]:
```suggestion
- [Install or update Azure CLI to version 2.25.0 and above](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest). Use the below command to check your current installed version.
```

File: azure_arc_servers_jumpstart/ssh/enable_ssh.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+export subscription=<your subscription ID>
+export resourcegroup=<your Resource Group where your Azure Arc-enabled server is registered to>
+export arc_server=<your Azure Arc-enabled server>
+
+## Main Script -- Do not change
+
+# Add Azure CLI extensions
+
+    echo  "Adding Azure CLI extensions"
+    az extension add --name ssh
```
Comment (likamrat) [inline azure_arc_servers_jumpstart/ssh/enable_ssh.sh:14]:
```suggestion
echo  "Adding Azure CLI extensions"
az extension add --name ssh
```

Comment (likamrat) [reply azure_arc_servers_jumpstart/ssh/enable_ssh.sh:14]: will this work without _sudo_?

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
+---
+type: docs
+title: "SSH Access"
+linkTitle: "SSH Access"
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md:4]:
```suggestion
title: "Server SSH Access"
linkTitle: "Server SSH Access"
```

File: azure_arc_servers_jumpstart/ssh/enable_ssh.sh [hunk 4]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Environment variables edit to match your enviroment
+
+export subscription=<your subscription ID>
+export resourcegroup=<your Resource Group where your Azure Arc-enabled server is registered to>
+export arc_server=<your Azure Arc-enabled server>
```
Comment (likamrat) [inline azure_arc_servers_jumpstart/ssh/enable_ssh.sh:7]:
```suggestion
export resourceGroup=<your Resource Group where your Azure Arc-enabled server is registered to>
export arcServer=<your Azure Arc-enabled server>
```

Comment (likamrat) [reply azure_arc_servers_jumpstart/ssh/enable_ssh.sh:7]: update screenshot as well

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+4. User tests SSH connection.
+
+## Deployment
+
+- To run the automation, navigate to the [deployment folder](https://github.com/microsoft/azure_arc/tree/main/azure_arc_servers_jumpstart/ssh) and edit the environment section of the [shell script](https://github.com/microsoft/azure_arc/tree/main/azure_arc_servers_jumpstart/ssh/enable_ssh.sh)
+
+  - _`subscription`_: your Azure subscription ID.
+  - _`resourcegroup`_: Resource Group where your Azure Arc-enabled server is registered to.
+  - _`arc_server`_: Name of your Azure Arc-enabled server as it is shown in the Azure Portal.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md:106]:
```suggestion
  - _`subscription`_: your Azure subscription ID.
  - _`resourceGroup`_: Resource Group where your Azure Arc-enabled server is registered to.
  - _`arcServer`_: Name of your Azure Arc-enabled server as it is shown in the Azure Portal.
```

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+  ```shell
+    . ./enable_ssh.sh
+  ```
+
+  > **NOTE: The extra dot is due to the script having an _export_ function and needs to have the vars exported in the same shell session as the other commands.**
+
+    ![Script's output](./02.png)
+
+- Now it is necessary to connect to the Azure Arc-enabled server and allow connections on the Connected Machine agent, to do that connect to the server and open an administrative session and run the command:
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md:120]:
```suggestion
- It is required to connect to the Azure Arc-enabled server for allowing connectivity to the Connected Machine agent. To do that, connect to the server and open an administrative session and run the below command.
```

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+
+  ```shell
+    azcmagent config set incomingconnections.ports 22
+  ```
+
+    ![Set Agents incomming connections](./03.png)
+
+## SSH connection test
+
+- Once the script has finished its run, you should be able to SSH to the Azure Arc-enabled server by running:
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md:130]:
```suggestion
- Once the script has finished its run, you should be able to connect via SSH to the Azure Arc-enabled server by running:
```

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+
+  ```shell
+    az ssh arc --resource-group <resource_group> --name <arc_enabled_server> --local-user <username>
+  ```
+
+    ![Test SSH connection](./04.png)
+
+  > **NOTE: if your Arc-enabled server uses other authentication methods make sure to review the [az ssh arc documentation](https://docs.microsoft.com/en-us/cli/azure/ssh?view=azure-cli-latest#az-ssh-arc)**
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md:138]:
```suggestion
  > **NOTE: if your Arc-enabled server uses other authentication methods make sure to review the [az ssh arc documentation](https://docs.microsoft.com/cli/azure/ssh?view=azure-cli-latest#az-ssh-arc)**
```

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+
+## Cleanup
+
+To disable SSH access to an Azure Arc-enabled server, run the the command:
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md:142]:
```suggestion
To disable SSH access to an Azure Arc-enabled server, run the below command.
```

File: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+
+  ```shell
+  az rest --method delete --uri https://management.azure.com/subscriptions/<subscription>/resourceGroups/<resourcegroup>/providers/Microsoft.HybridCompute/machines/<arc enabled server name>/providers/Microsoft.HybridConnectivity/endpoints/default?api-version=2021-10-06-preview
+  ```
+
+On the Azure Arc enabled-server's operating system, remove the configured port by running the command below on the VM:
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_ssh/_index.md:148]:
```suggestion
On the Azure Arc enabled-server's operating system, remove the configured port by running the below command on the server:
```

**Review by likamrat — CHANGES_REQUESTED** [review]
- _sudo_ related comment
- parameters name
- lingo and syntax

**Comment by lanicolas** [comment]
applied the changes and updated screenshot

**Pull request merged**